policy/modules/admin/bootloader.te

```diff
@@ -259,8 +259,3 @@ optional_policy(`
 optional_policy(`
 	rpm_rw_pipes(bootloader_t)
 ')
-
-ifdef(`distro_gentoo',`
-	# Fix bug #537652 - grub2-mkconfig has search rights needed on current dir (usually user home dir)
-	userdom_search_user_home_dirs(bootloader_t)
-')
```

Comment:
Does `grub-mkconfig -o /boot/grub/grub.cfg` continue to work after this is removed? I don't use grub anymore so I can't readily test this.

Reply:
Nevermind, I see you pointed this out in your original comment. :)